web-components/src/components/tabs/Tab.ts

```diff
@@ -87,22 +114,33 @@ export namespace Tab {
       }
     }
 
-    handleKeyDown(event: KeyboardEvent) {
+    handleCrossClick(event: MouseEvent) {
+      event.preventDefault();
+      if (this.disabled === true) return;
       if (this.id) {
+        if(this.closable === "auto"){
         this.dispatchEvent(
-          new CustomEvent<TabKeyDownEvent>("tab-keydown", {
+          new CustomEvent<TabClickEvent>("tab-cross-click", {
             detail: {
-              id: this.id,
-              key: event.code,
-              ctrlKey: event.ctrlKey,
-              shiftKey: event.shiftKey,
-              altKey: event.altKey,
-              srcEvent: event
```

Comment:
Removing the KeyDown handling seems like an accessibility problem. How is this same functionality re-instated?

Reply:
The keydown has been moved to the parent component md-tabs and the event that will be received outside will be exactly the same.